t/api/14-plugin_obs_rsync.t

```diff
@@ -9,6 +9,9 @@ use OpenQA::Test::TimeLimit '50';
 use Mojo::IOLoop;
 use OpenQA::Test::Utils 'perform_minion_jobs';
 use OpenQA::Test::ObsRsync 'setup_obs_rsync_test';
+use OpenQA::WebAPI::Plugin::ObsRsync qw(register);
+use File::Temp qw(tempdir);
+use Mojo::File qw(path);
```

Comment:
These additions seem unrelated as no other lines in that file were changed.

Reply:
I forgot to remove the imports when replace the test with the uncoverable statement